Things.md

```diff
@@ -0,0 +1,117 @@
+This is a general guide about Things for all FullScreenShenanigans projects. This guide will show you how in-game Things are stored and manipulated. Everything you see in the game (trees, blocks, the player, etc.) is a Thing.  
+The Thing class is subclassed by a new class for everything (Tree class, Block class, Player class, etc.). These Things can be manipulated to change their location and properties.  
+All the following code samples are written specific to `FSM` (FullScreenMario) but work for all FullScreenShenanigans projects (`FSM`, `FSP`, etc.).  
+  
+### Table of Contents  
+1. [GroupHoldr](#groupholdr)
+    1. [Getting Groups](#getting-groups)
+2. [Manipulating Things](#manipulating-things)
+    1. [Shifting Things](#shifting-things)
+    2. [Opacity](#opacity)
+
+# GroupHoldr  
+
+GroupHoldr is a module that's used to store similarly typed information, such as Things, based on a shared property. Each Thing has a `groupType` string property that determines what group it's considered to be in. For example, the groups in `FSM` are:
+
+* Text
+* Character
+* Solid
+* Scenery 
+
+GroupHoldr contains an Array for each of the groups; each Array contains all the Things of that type currently in the game. Things are added to their respective group when added to the game, and removed when they die. For example, in `FSM`, solid objects like floors and blocks would be stored in the Solid group. 
```

Comment:
If you remove the comma from "game, and removed" it makes the paragraph flow better.